doc/rados/configuration/storage-devices.rst

```diff
@@ -2,22 +2,26 @@
  Storage Devices
 =================
 
-There are two Ceph daemons that store data on devices:
+There are several Ceph daemons that store data on devices:
```

Comment:
the mgr doesn't store any data on a local device, so this sentence should be rewritten. Maybe something like "There are several Ceph daemons in a storage cluster:"

Reply:
This sentence has been updated to read the way you've written it.